Adding logging broke GitHub actions tests

Bad Apps: Blogs is passing all tests in the development environment, but not in GitHub actions.

It looks like the tests that are failing have to do with logging in. There is a similar failed assertion in all of these.

```
=========================== short test summary info ============================
FAILED tests/test_auth.py::test_register - AssertionError: assert 'http://loc.../
FAILED tests/test_auth.py::test_login - AssertionError: assert '/' == 'http:/...
FAILED tests/test_blog.py::test_login_required[/create] - AssertionError: ass...
FAILED tests/test_blog.py::test_login_required[/1/update] - AssertionError: a...
FAILED tests/test_blog.py::test_login_required[/1/delete] - AssertionError: a...
FAILED tests/test_blog.py::test_delete - AssertionError: assert '/' == 'http:...
========================= 6 failed, 21 passed in 2.72s =========================
```

The tests expect 'http://localhost/' as a prefix to the resource name, but the response only has the resource name. For example:

```
________________________________ test_register _________________________________

client = <FlaskClient <Flask 'bad_apps_blog'>>, app = <Flask 'bad_apps_blog'>

    def test_register(client, app):
        assert client.get('/auth/register').status_code == 200
        response = client.post(
            '/auth/register', data={'username': 'a', 'displayname' : 'user1' ,'password': 'a'}
        )
>       assert 'http://localhost/auth/login' == response.headers['Location']
E       AssertionError: assert 'http://localhost/auth/login' == '/auth/login'
E         - /auth/login
E         + http://localhost/auth/login
```

--- a/tests/test_auth.py
+++ b/tests/test_auth.py
@@ -8,7 +8,7 @@ def test_register(client, app):
     response = client.post(
         '/auth/register', data={'username': 'a', 'displayname' : 'user1' ,'password': 'a'}
     )
-    assert 'http://localhost/auth/login' == response.headers['Location']
+    assert '/auth/login' == response.headers['Location']
 
     with app.app_context():
         assert get_db().execute(
@@ -34,7 +34,7 @@ def test_register_validate_input(client, username, displayname, password, messag
 def test_login(client, auth):
     assert client.get('/auth/login').status_code == 200
     response = auth.login()
-    assert response.headers['Location'] == 'http://localhost/'
+    assert response.headers['Location'] == '/'
 
     with client:
         client.get('/')

--- a/tests/test_blog.py
+++ b/tests/test_blog.py
@@ -23,7 +23,7 @@ def test_index(client, auth):
 ))
 def test_login_required(client, path):
     response = client.post(path)
-    assert response.headers['Location'] == 'http://localhost/auth/login'
+    assert response.headers['Location'] == '/auth/login'
 
 
 def test_author_required(app, client, auth):
@@ -84,7 +84,7 @@ def test_create_update_validate(client, auth, path):
 def test_delete(client, auth, app):
     auth.login()
     response = client.post('/1/delete')
-    assert response.headers['Location'] == 'http://localhost/'
+    assert response.headers['Location'] == '/'
 
     with app.app_context():
         db = get_db()

